package.json

```diff
@@ -9,5 +9,7 @@
     "next-cloudinary",
     "scripts"
   ],
-  "devDependencies": {}
+  "dependencies": {
+    "react": "^18.2.0"
```

Comment:
was this an intentional add? dont think this is needed here 🤔 along with that i dont think the package-lock and yarn.lock changes are then related, can you revert those?